homeassistant/__init__.py

```diff
@@ -27,7 +27,7 @@
 DOMAIN = "homeassistant"
 
 # How often time_changed event should fire
-TIMER_INTERVAL = 10  # seconds
+TIMER_INTERVAL = 3  # seconds
```

Comment:
Oh, this one is dangerous, there are a bunch of components that listen to each time changed events to update their state. I'll make sure I'll update them.